### PR TITLE
Update documentation for IS 7.0.0 onwards 

### PR DESCRIPTION
**Pull request by shashimalcse**
## Related Issues
- https://github.com/wso2/product-is/issues/21127

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This page provides instructions on how to configure the LinkedIn
 authenticator and the WSO2 Identity Server using a sample app to 
 demonstrate authentication.
 
+> For WSO2 Identity Server 7.0.0 and later versions, please refer to the [updated documentation](is7/README.md) for the latest configuration process.
```
Comment (ImalshaG) [inline docs/README.md]: Shall we keep the IS 7.0.0 onwards docs in the main readme and place the older version docs at an inner folder?

File: docs/is7/img/linkedin-sign-in-page.png
Comment (ImalshaG) [inline docs/is7/img/linkedin-sign-in-page.png:1]: Is it possible to make this image smaller?

File: docs/is7/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+1. Download the .jar file from the [WSO2 Store](https://store.wso2.com/connector/identity-outbound-auth-linkedIn)
+    - **OAuth 2.0** : `org.wso2.carbon.extension.identity.authenticator.linkedin.connector-x.x.x` 
+    - **OpenID Connect (V2)** : `org.wso2.carbon.extension.identity.authenticator.linkedin.oidc.connector-x.x.x`
+2. Place the authenticator.jar file into the 
+`<IS_HOME>/repository/components/dropins` directory. 
+
+3. Start or restart the WSO2 Identity Server. 
+
+> If you want to upgrade the LinkedIn (.jar) in your existing Identity Server pack, please refer [upgrade instructions](https://docs.wso2.com/display/ISCONNECTORS/Upgrading+an+Authenticator).
```
Comment (ImalshaG) [inline docs/is7/README.md]: Is this link working?

File: docs/is7/README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+## Register WSO2 Identity Server on LinkedIn
+
+1. Navigate to LinkedIn [Create an app page](https://www.linkedin.com/developer/apps/new).
+
+2. Enter the required details.
+    * App Name and Company details. The company should have a LinkedIn page.
+    * Upload an image that you wish to use as the company logo.
+    * Select the checkbox to agree to the LinkedIn terms and conditions.
+3. Click on **Create App**. You will be redirect to a page with **Settings** for the created application.
```
Comment (ImalshaG) [inline docs/is7/README.md]:
```suggestion
3. Click on **Create App**. You will be redirected to the  **Settings** page of the created application. 
```

File: docs/README.md (hunk 1 above)
Comment (shashimalcse) [reply docs/README.md]: fixed

File: docs/is7/img/linkedin-sign-in-page.png
Comment (shashimalcse) [reply docs/is7/img/linkedin-sign-in-page.png:1]: fixed

File: docs/is7/README.md (hunk 2 above)
Comment (shashimalcse) [reply docs/is7/README.md]: fixed

File: docs/is7/README.md (hunk 3 above)
Comment (shashimalcse) [reply docs/is7/README.md]: fixed

**Review by ImalshaG — APPROVED** [review]

**Pull request merged**